packages/flutter_tools/templates/app/linux.tmpl/runner/my_application.cc.tmpl

```diff
@@ -15,8 +15,7 @@ struct _MyApplication {
 G_DEFINE_TYPE(MyApplication, my_application, GTK_TYPE_APPLICATION)
 
 // Called when first Flutter frame received.
-static void first_frame_cb(MyApplication* self, FlView *view)
-{
+static void first_frame_cb(MyApplication* self, FlView *view) {
```

Comment:
Do you want to fix the pointer alignment while you are fixing style on this line? `FLView *view` appears to be the only thing in the file using a right-aligned pointer.

Reply:
I ran it though clang-format and they should have all been left aligned. A bigger change but probably a better long term fix.

Reply:
I ran clang-format from the project root, which didn't pick up the config. Re-run.